test/testData/simple_remap.ttl

```diff
@@ -0,0 +1,9 @@
+@prefix bao:   <http://www.bioassayontology.org/bao#> .
+@prefix bat:   <http://www.bioassayontology.org/bat#> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix obo:   <http://purl.obolibrary.org/obo/> .
+
+obo:CHEBI_101086 bat:remapTo obo:CHEBI_102248 .
```

Comment:
I would suggest making a really basic tree, e.g.

A
-> B
-> C

... and first making sure it has those 3 things in the given hierarchy. Then have a fork of the file that remaps B to C, and make sure that the branch now looks like:

A
-> C

That would get the ball rolling, and we can add the rest of the necessary tests one at a time.